src/buttons/test/buttons.spec.js

```diff
@@ -201,7 +201,7 @@ describe('buttons', function () {
         expect(btns.eq(1)).not.toHaveClass('active');
 
         btns.eq(0).click();
-        expect($scope.model).toEqual(undefined);
+        expect($scope.model).toBeNull();
```

Comment:
IIRC, `toEqual` is the equivalent of `==`, so it could just be a not specific enough test.